### PR TITLE
Fix possible to start installation without network (#1221109)

### PR DESCRIPTION
**Pull request by jkonecny12**
Payload manager is started from network spoke if network settings change.
This will start source spoke and software spoke reach-ability testing.

*Resolves: rhbz#1221109*

### CODE REVIEW

File: pyanaconda/ui/gui/spokes/network.py [hunk 1]
```diff
@@ -1418,12 +1419,19 @@ def __init__(self, *args, **kwargs):
...
+        # true if network settings change (hostname excluded)
+        self._network_change = False
 
     def apply(self):
         _update_network_data(self.data, self.network_control_box)
         log.debug("network: apply ksdata %s", self.data.network)
         self.network_control_box.kill_nmce(msg="leaving network spoke")
 
+        # if installation media is not used try if source is reachable
+        if self.data.method.method != "cdrom" and self._network_change:
```
Comment (bcl) [inline pyanaconda/ui/gui/spokes/network.py]: hd is another possible non-network source.

Comment (jkonecny12) [inline pyanaconda/ui/gui/spokes/network.py]: True. I'll change it locally.

**Comment by jkonecny12** [comment]
Add check for *harddrive* too. 
Thank you @bcl .

**Comment by clumens** [comment]
Note that on this PR, the jenkins results are correct (even if it tested too much - don't know what's up with that).  There are some pylint problems.

**Comment by jkonecny12** [comment]
Yeah I already fixed it. Thank you for this PR testing @clumens .

**Comment by vojtechtrefny** [comment]
Looks good.

**Comment by jkonecny12** [comment]
Pushed. Thank you

**Pull request closed without merging**